examples/AspNetCore/ControllerSample/CustomTopicAttribute.cs

```diff
@@ -18,12 +18,19 @@ public CustomTopicAttribute(string pubsubName, string name)
         {
             this.PubsubName = Environment.ExpandEnvironmentVariables(pubsubName);
             this.Name = Environment.ExpandEnvironmentVariables(name);
+            this.Priority = int.MaxValue;
```

Comment:
Suggestion - just let this be zero by default. This lets people increase and decrease priority relative to default. It's also a little less surprising that an int field has a default value of zero.